.opencode/commands/resolve-review.md

```
@@ -0,0 +1,87 @@
+    }
+  }
+}' -F owner="$OWNER" -F repo="$REPO" -F number="$PR_NUMBER" \
+  | jq '[.data.repository.pullRequest.reviewThreads.nodes[] | select(.isResolved == false)]'
+```
+
+## 3. Address Each Comment
+
+Read `diffHunk` (code context) and `body` (feedback):
+
+| Situation | Action |
+|-----------|--------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)